src/pages/layer-2.js

```diff
@@ -0,0 +1,618 @@
+// Libraries
+import React from "react"
+import { graphql } from "gatsby"
+import { getImage } from "gatsby-plugin-image"
+import styled from "styled-components"
+import { useIntl } from "gatsby-plugin-intl"
+
+// Assets
+import privacyGlyph from "../assets/run-a-node/privacy-glyph.svg"
+
+// Data
+import cexSupport from "../data/layer-2/cex-layer-2-support.json"
+import layer2Data from "../data/layer-2/layer-2.json"
+import validiumData from "../data/validium.json"
+
+// Components
+import ExpandableCard from "../components/ExpandableCard"
+import InfoBanner from "../components/InfoBanner"
+import Link from "../components/Link"
+import PageHero from "../components/PageHero"
+import PageMetadata from "../components/PageMetadata"
+import Pill from "../components/Pill"
+import ProductCard from "../components/ProductCard"
+import { CardGrid, Content, Page } from "../components/SharedStyledComponents"
+
+// Utils
+import { translateMessageId } from "../utils/translations"
+
+// Styles
+const GappedPage = styled(Page)`
+  gap: 4rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    gap: 3rem;
+  }
+  * {
+    scroll-margin-top: 5.5rem;
+  }
+`
+
+const HeroContainer = styled.div`
+  background: ${({ theme }) => theme.colors.runNodeGradient};
+  width: 100%;
+`
+
+const Hero = styled(PageHero)`
+  padding-bottom: 2rem;
+`
+
+const Flex50 = styled.div`
+  flex: 50%;
+  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
+    flex: 100%;
+  }
+`
+
+const TwoColumnContent = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 2rem;
+  gap: 2rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.l}) {
+    flex-direction: column;
+    align-items: flex-start;
+    margin-left: 0rem;
+    margin-right: 0rem;
+  }
+`
+
+const InfoGrid = styled(CardGrid)`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(min(100%, 340px), 1fr));
+  gap: 1rem 2rem;
+  & > div {
+    height: fit-content;
+    &:hover {
+      transition: 0.1s;
+      transform: scale(1.01);
+      img {
+        transition: 0.1s;
+        transform: scale(1.1);
+      }
+    }
+  }
+`
+
+const StyledCardGrid = styled(CardGrid)`
+  margin-bottom: 4rem;
+  margin-top: 4rem;
+`
+
+const Layer2Page = ({ data }) => {
+  const intl = useIntl()
+  const heroContent = {
+    title: "Layer 2",
+    header: "Layer 2",
+    subtitle: "Smart solutions to scale Ethereum",
+    image: getImage(data.ethBlocks),
+    alt: "test",
+    buttons: [
+      {
+        content: "Use Layer 2",
+        pathId: "use-layer-2",
+      },
+      {
+        content: "Wait, what's Ethereum?",
+        pathId: "/",
+        isSecondary: "isSecondary",
+      },
+    ],
+  }
+
+  const layer2Cards = [
```

Comment:
I like these card but I'm not sure they need to be expandable/collapsable. I'd push us to make the copy succinct enough that they can just be standard cards, like in [ETH page](https://ethereum.org/en/eth/):
![Image 2022-03-13 at 6 42 03 PM](https://user-images.githubusercontent.com/8097623/158090662-672b36f1-1581-4ea5-8faa-23d4cb162b63.jpg)